parsely/src/main/java/com/parsely/parselyandroid/ParselyTracker.java

```diff
@@ -75,16 +76,20 @@ protected ParselyTracker(String siteId, int flushInterval, Context c) {
         context = c.getApplicationContext();
         eventsBuilder = new EventsBuilder(context, siteId);
         localStorageRepository = new LocalStorageRepository(context);
+        flushManager = new FlushManager(this, flushInterval * 1000L,
+                ParselyCoroutineScopeKt.getSdkScope());
+        inMemoryBuffer = new InMemoryBuffer(ParselyCoroutineScopeKt.getSdkScope(), localStorageRepository, () -> {
+            if (!flushTimerIsActive()) {
```

Comment:
Praise (❤️): Love the listener addition, make the whole implementation much more simpler and at the same time harder to mess with, kudos! 🥇